astropy/io/votable/tests/table_test.py

```diff
@@ -175,6 +179,80 @@ def test_write_with_format():
 def test_empty_table():
     votable = parse(
         get_pkg_data_filename('data/empty_table.xml'),
-        pedantic=False)
+        verify='warn')
     table = votable.get_first_table()
     astropy_table = table.to_table()  # noqa
+
+
+def test_verify_options(tmpdir):
+
+    # Start off by checking the default (ignore)
+
+    with catch_warnings(VOWarning) as w:
+        parse(get_pkg_data_filename('data/gemini.xml'))
+    assert len(w) == 0
+
+    # Then try the various explicit options
+
+    with catch_warnings(VOWarning) as w:
+        parse(get_pkg_data_filename('data/gemini.xml'), verify='ignore')
+    assert len(w) == 0
+
+    with catch_warnings(VOWarning) as w:
+        parse(get_pkg_data_filename('data/gemini.xml'), verify='warn')
+    assert len(w) == 25
+
+    with pytest.raises(VOWarning):
+        parse(get_pkg_data_filename('data/gemini.xml'), verify='exception')
+
+    # Make sure the pedantic option still works for now (pending deprecation)
+
+    with catch_warnings(VOWarning) as w:
+        parse(get_pkg_data_filename('data/gemini.xml'), pedantic=False)
+    assert len(w) == 25
```

Comment:
For the `pedantic` backward-compat test, we should also check that `AstropyDeprecationWarning` is not *yet* raised. This way, when we remove `pending=True`, the tests will fail and we will have to update the test to catch the deprecation warning, and the warning wouldn't "leak through" in a way that `pytest` would complain ungracefully about it.

Reply:
Done